conda/recipes/libcuspatial/meta.yaml

```
@@ -86,12 +106,24 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
```

Do we need `ninja` here too (as it is used elsewhere)?

```suggestion
        - cmake {{ cmake_version }}
        - ninja
```

---

No, we do not need ninja here. Ninja should be in the top-level `build` but not the subpackages' `build` sections.